packages/feedback-spa/package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng build --base-href /feedback/",
+    "build": "ng build --prod --base-href /feedback/",
```

Comment:
```suggestion
    "build": "ng build --production --base-href /feedback/",
```

Reply:
@hybridx Should it be --production here in the build command?

Reply:
This is for the ng cli.
Here's the documentation link I referred `https://angular.io/cli/build`

Reply:
This looks right